### PR TITLE
Add app groups

### PR DESCRIPTION
**Pull request by m4rk055**
Solves the issue with [one client handling multiple applications](https://github.com/blindnet-io/clients-management/issues/70#issue-1421431169).
Apps in the same group use the same key to sign token.

Also adds docker-compose to start the dev environment and refactors the `Env` class.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pg_user=identity
+pg_pass=identity
+pg_db_name=identity
+pg_port_host=5433
```
Comment (TheKinrar) [inline .env]: 5433? We should probably default to Postgres default port
(found in multiple files of the PR)

Comment (TheKinrar) [reply .env]: Also, what is this file used for? The env names do not match the ones used by the server

File: src/main/scala/db/Migrator.scala [hunk 2]
```diff
@@ -5,11 +5,15 @@ import cats.effect.IO
...
 object Migrator {
-  def migrate(): IO[Unit] = IO {
-    Flyway.configure()
-      .dataSource(Env.get.dbUri, Env.get.dbUsername, Env.get.dbPassword)
+  def migrate(env: Env): IO[Unit] = IO {
+
+    Flyway
+      .configure()
+      .dataSource(env.dbUri, env.dbUsername, env.dbPassword)
+      .table("Flyway")
```
Comment (TheKinrar) [inline src/main/scala/db/Migrator.scala]: changing this will break our existing deployment and is useless as far as I can tell

**Review by TheKinrar — CHANGES_REQUESTED** [review]
Let's say A is a DPO company, B and C are clients of A, handling personal data.
This of course means A has access to B and C, but also that B has access to C, and I'm not sure we want that.

**Comment by m4rk055** [comment]
> Let's say A is a DPO company, B and C are clients of A, handling personal data. This of course means A has access to B and C, but also that B has access to C, and I'm not sure we want that.

A is the only one holding the secret key and issuing tokens. Users of B will be issued tokens for B app and won't be authorized to access data belonging to C app.

**Comment by TheKinrar** [comment]
> A is the only one holding the secret key and issuing tokens.

How does this work? B and C might manage their own infrastructure, have their own servers, etc. I feel like the scenario where B and C never have access to the private key will not fit many use cases

File: .env (hunk 1 above)
Comment (m4rk055) [reply .env]: It's used by docker-compose

**Review by TheKinrar — APPROVED** [review]

**Pull request merged**